Gifski/util.swift

```diff
@@ -938,11 +938,59 @@ extension NSView {
 		])
 	}
 
+	func centerX(inView view: NSView) {
+		translatesAutoresizingMaskIntoConstraints = false
+
+		NSLayoutConstraint.activate([
+			centerXAnchor.constraint(equalTo: view.centerXAnchor)
+		])
+	}
+
+	func centerY(inView view: NSView) {
+		translatesAutoresizingMaskIntoConstraints = false
+
+		NSLayoutConstraint.activate([
+			centerYAnchor.constraint(equalTo: view.centerYAnchor)
+		])
+	}
+
 	func addSubviewToCenter(_ view: NSView) {
 		addSubview(view)
 		view.center(inView: superview!)
 	}
 
+	func constraintTopBottom(bottomView view: NSView) {
+		translatesAutoresizingMaskIntoConstraints = false
+
+		NSLayoutConstraint.activate([
+			topAnchor.constraint(greaterThanOrEqualTo: view.bottomAnchor)
+		])
+	}
+
+	func constraintBottomTop(topView view: NSView) {
+		translatesAutoresizingMaskIntoConstraints = false
+
+		NSLayoutConstraint.activate([
+			bottomAnchor.constraint(lessThanOrEqualTo: view.topAnchor)
+		])
+	}
+
+	func constraintTopConstant(withView view: NSView, constant: CGFloat = 0) {
```

Comment:
The naming could be improved here too, and it could be more general. `constrainToEdge(.top, padding: 10)` where `padding` is optional and defaults to `0`.

`CGFloat` => `Double`

I think you could use `NSEdgeInsets`.

Reply:
I changed it and made it more generic. We can use this function to match horizontal or vertical edges with a given padding.